src/rc/start-stop-daemon.c

```diff
@@ -712,6 +712,17 @@ start_stop_daemon(int argc, char **argv)
 		if (sscanf(tmp, "%d", &nicelevel) != 1)
 			eerror("%s: invalid nice level `%s' (SSD_NICELEVEL)",
 			    applet, tmp);
+		if ((tmp = getenv("SSD_IONICELEVEL"))) {
```

Comment:
Oh?

I'm fairly sure that I'm following StyleGuide.md here. Notice that the preceding if (712) is indented with two tabs, the eerror statement that follows it (713) is indented with three.

What have I done wrong?